documentation/connect_puppet_master.markdown

```diff
@@ -16,77 +16,76 @@ canonical: "/puppetdb/latest/connect_puppet_master.html"
 [report_format]: ./api/wire_format/report_format_v5.html
 [puppetdb_server_urls]: ./puppetdb_connection.html#server_urls
 
-> Note: To use PuppetDB, your site's Puppet master(s) must be running Puppet 4.0.0 or later.
+> Note: To use PuppetDB, your site's Puppet master(s) must be running Puppet version 4.0.0 or later.
 
-After PuppetDB is installed and running, you should configure your Puppet master(s) to use it. Once connected to PuppetDB, Puppet masters will do the following:
+Once PuppetDB is installed and running, configure your Puppet master(s) to use it. When properly connected to PuppetDB, Puppet masters will do the following:
 
 * Send every node's catalog to PuppetDB
 * Send every node's facts to PuppetDB
```

Comment:
reports should have a line here too. Maybe we should just have one bullet for the three.